src/state/effects/types.ts

```diff
@@ -11,9 +11,15 @@ type VisibleEffectProps = {
 };
 export type EffectDescription = InvisibleEffectProps | VisibleEffectProps;
 export type BaseEffect = {
+  userStoryChange?: number;
   gremlinChange?: number;
   capacityChange?: number;
-  userStoryChange?: number;
+  /** @deprecated this prop does not exist, use userStoryChange */
+  userStoryChance?: never;
+  /** @deprecated this prop does not exist, use gremlinChange */
+  gremlinChance?: never;
+  /** @deprecated this prop does not exist, use capacityChange */
+  capacityChance?: never;
```

Comment:
This will cause typescript to fail if the above typo happens again, but wont prevent `userStoryChagne` or similar.

Reply:
use of @deprecated feels weak, since there an infinite number of misspellings I could make. The explicit return types fpr effect factory seems more elegant. I don't quite think I understand how that works.